services/talk/service.go

```diff
@@ -0,0 +1,63 @@
+package talk
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"log"
+	"net/http"
+
+	"github.com/influxdata/kapacitor"
+)
+
+type Service struct {
+	url        string
+	authorName string
+	logger     *log.Logger
+}
+
+func NewService(c Config, l *log.Logger) *Service {
+	return &Service{
+		url:        c.URL,
+		authorName: c.AuthorName,
+		logger:     l,
+	}
+}
+
+func (s *Service) Open() error {
+	return nil
+}
+
+func (s *Service) Close() error {
+	return nil
+}
+
+func (s *Service) Alert(title, text string, level kapacitor.AlertLevel) error {
```

Comment:
Looks like the `level` parameter is unused. If we don't need it let's remove it.